### DISCUSSION TITLE
Helix does not fill in the entire window background

### DISCUSSION THREAD

**Discussion by pathwave** [helix-editor/helix#4236] [Q&A] [answered]
Helix does not fill in the entire window background and leaves some undesired lines on the bottom and side of the terminal.

Could probably paint the entire window once on start and on resize.

**Comment by the-mikedavis** [comment, marked as answer]
Helix paints the available background. The edges you see are borders in the terminal emulator, usually drawn as padding for the terminal area. You may be able to configure the size of the borders depending on the terminal emulator. Theming the terminal emulator's background color to match the background color in the Helix theme you use can make this visually disappear.

**Reply by pathwave** [reply]
I did some research, seems like we are limited to drawing in text size sadly.
